src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -1009,7 +1008,8 @@ def call(
             total_loss = self.compute_loss(labels=d_labels, logits=(prediction_scores, seq_relationship_score))
 
         if not inputs["return_dict"]:
-            return (prediction_scores, seq_relationship_score) + outputs[2:]
+            output = (prediction_scores, seq_relationship_score) + outputs[2:]
+            return ((total_loss,) + output) if total_loss is not None else output
```

Comment:
It was not returning the loss before that?

Reply:
This should have been caught by a test

Reply:
The test on the loss computation was working because the test is run only for `return_dict=True`, it would have failed if `return_dict=False`.